tests/model_explainability/guardrails/test_guardrails.py

```
@@ -45,6 +46,33 @@
 HAP_DETECTOR: str = "hap-detector"
 
 
+def check_unsuitable_inputs_for_prompts(
+    current_client_token: str,
+    guardrails_orchestrator_route: Route,
+    openshift_ca_bundle_file: str,
+    prompts: List[GuardrailsDetectionPrompt],
+):
+    for prompt in prompts:
+        response = requests.post(
```

Ruff (0.14.0)

Probable use of `requests` call without timeout

(S113)